ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Arguments.java

```diff
@@ -537,295 +315,195 @@ private void loadProperties() {
     }
 
     private void printVersion() {
-        System.out.println("Version: " + this.getVersion());
-        System.out.println("Build Date: " + this.getBuildDate());
+        System.out.println("Version: " + this.version);
```

Comment:
With pulling the behavior out of the argument parsing logic this code should be relocated as well.